__tests__/app.test.js

```diff
@@ -48,4 +48,25 @@ describe('/api (getting all endpoints)', () => {
         });
     });
 });
+
+describe('Get all articles', () => {
+    test('making sure to return all articles', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((articles) => {
+            expect(articles.body.articles[0])
```

Comment:
We should test every element in the array, we can use a loop and then objectContaining and expect every value to be any "type" 🙂